### PR TITLE
Add map component

### PR DESCRIPTION
**Pull request by l2fprod**
Show google maps in Dashboard together with the "raw data" view - to be removed on next iteration of the ui.

Open question:
How to have the Dashboard take the full height so that I don't have to set a height: 600px but rather a height: 100%?

### CODE REVIEW

**Comment by rvennam** [comment]
LGTM

File: src/routes/Dashboard/components/Map/Map.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Map.propTypes = {
+  center: React.PropTypes.array,
+  zoom: React.PropTypes.number,
+  distributionCenters: React.PropTypes.array,
+  shipments: React.PropTypes.array,
+  retailers: React.PropTypes.array,
+};
+
+Map.defaultProps = {
```
Comment (colbycheeze) [inline src/routes/Dashboard/components/Map/Map.jsx]: Side Note: This is something we can move to "initialState" in the dashboard module. I never defined how the data should look in redux, I just dumped it in and called it "dbdata" for now, but we can set up this now that we understand how we need the data to come into the map component.

File: src/routes/Dashboard/components/Map/Map.style.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+[{
```
Comment (colbycheeze) [inline src/routes/Dashboard/components/Map/Map.style.json]: Any reason for storing the styles as json?

Comment (l2fprod) [inline src/routes/Dashboard/components/Map/Map.style.json]: What would be the alternative? It is the style attribute of the Google Maps object. Having it in the code itself was not good for readability. This makes it easy to replace with another style.

**Pull request merged**